.eslintrc

```diff
@@ -88,29 +88,34 @@
         "ignoreDestructuring": false
       }
     ],
-    "no-restricted-imports": ["error", { "paths": [{
-      "name": "styled-system",
-      "importNames": ["color"],
-      "message": "Please use the default export from 'src/style/utils/color' instead."
-      }]
-    }]
+    "no-restricted-imports": [
+      "error",
+      {
+        "paths": [
+          {
+            "name": "styled-system",
+            "importNames": ["color"],
+            "message": "Please use the default export from 'src/style/utils/color' instead."
+          }
+        ]
+      }
+    ]
   },
   "env": {
     "es6": true,
     "browser": true,
-    "jasmine": true
+    "jasmine": true,
+    "cypress/globals": true
   },
-  "extends": [
-    "airbnb",
-    "prettier",
-    "prettier/react"
-  ],
-  "plugins": ["react", "react-hooks"],
+  "extends": ["airbnb", "prettier", "prettier/react"],
+  "plugins": ["cypress", "react", "react-hooks"],
   "globals": {
     "jest": false,
     "global": false,
     "process": false,
-    "test": false
+    "test": false,
+    "Given": true,
+    "When": true,
+    "Then": true
```

Comment:
I think we should add also
"And": true

Reply:
Why? It wasn't there previously, do we need it?